File: OrchidCore/src/main/java/com/eden/orchid/impl/themes/tags/StylesTag.java
package com.eden.orchid.impl.themes.tags;

import com.eden.orchid.api.compilers.TemplateTag;
import com.eden.orchid.api.options.annotations.Description;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
@Description(value = "Render all style tags to the page.", name = "Styles")
public final class StylesTag extends TemplateTag {

    @Inject
    public StylesTag() {
        super("styles", Type.Simple, true);
    }

    @Override
    public String[] parameters() {
        return new String[] {};
    }

}

File: OrchidCore/src/main/kotlin/com/eden/orchid/impl/themes/tags/StylesTag.kt
package com.eden.orchid.impl.themes.tags

import com.eden.orchid.api.compilers.TemplateTag
import com.eden.orchid.api.options.annotations.Description

@Description(value = "Render all style tags to the page.", name = "Styles")
class StylesTag : TemplateTag("styles", Type.Simple, true) {

    override fun parameters() = emptyArray<String>()

}
